src/pages/Designer/Designer.js

```diff
@@ -117,55 +117,111 @@ const Designer = () => (
           <BarChart size="large" color="TabularGreen" />
         </Box>
       </Box>
+      <ResponsiveContext.Consumer>
+        {(responsive) =>
+          responsive === 'large' || responsive === 'xlarge' ? (
+            <Stack>
```

Comment:
I'm wondering if should use Stack at all or just use a Box with background and place the context inside. And in order to achieve the offset of the tiles with the background, you can use a negative margin instead.
If you feel like it is too much work at this point, we can postpone this refactoring to after the showcase.